iai_donbot_description/launch/display.launch

```diff
@@ -1,22 +1,10 @@
 <launch>
-  <arg name="model" />
-  <arg name="gui" default="True" />
-  <include file="$(find iai_donbot_description)/launch/upload.launch" />
-  <node pkg="tf2_ros" type="static_transform_publisher" name="fake_localization" output="screen" args="0 0 0 0 0 0 map odom" />
-  <param
-    name="use_gui"
-    value="$(arg gui)" />
-  <node
-    name="joint_state_publisher"
-    pkg="joint_state_publisher"
-    type="joint_state_publisher" />
-  <node
-    name="robot_state_publisher"
-    pkg="robot_state_publisher"
-    type="state_publisher" />
-  <!--node
-    name="rviz"
-    pkg="rviz"
-    type="rviz"
-    args="-d $(find iai_boxy_base_2)/urdf.rviz" /-->
-</launch>
+	<arg name="model" />
+	<arg name="gui" default="True" />
+	<include file="$(find iai_donbot_description)/launch/upload.launch" />
+	<node pkg="tf2_ros" type="static_transform_publisher" name="fake_localization" output="screen" args="0 0 0 0 0 0 map odom" />
+	<param name="use_gui" value="$(arg gui)" />
+	<node name="joint_state_publisher_gui" pkg="joint_state_publisher_gui" type="joint_state_publisher_gui" />

```

Comment:
Why are you using joint_state_publisher_gui, instead of joint_state_publisher?

Reply:
joint_state_publisher is deprecated, it shows up warning that I dont like